{"name": "test_login_neg_01", "status": "passed", "description": "\nTC_ID: LOGIN_NEG_01\nTitle: Login with locked user\nExpected: Error message displayed\n", "attachments": [{"name": "screenshot_input_neg01", "source": "2bae5273-6dd7-4806-a02b-99ba1299d84d-attachment.png", "type": "image/png"}, {"name": "screenshot_result_neg01_as expected", "source": "85cbd556-da3d-490e-99d3-bfd456fcdc78-attachment.png", "type": "image/png"}], "start": 1759695345826, "stop": 1759695347741, "uuid": "74d24f56-35a7-49a1-b598-8a5a9a16ef81", "historyId": "e70bb41c655e04bab0dccb0e24676a48", "testCaseId": "e70bb41c655e04bab0dccb0e24676a48", "fullName": "tests.test_login#test_login_neg_01", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-N1PMKPL"}, {"name": "thread", "value": "26896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}